app/parse.py

```diff
@@ -15,8 +32,61 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def create_driver() -> WebDriver:
+    chrome_options = Options()
+    chrome_options.add_argument("--headless")
+    return webdriver.Chrome(options=chrome_options)
+
+
+def single_product(product: BeautifulSoup) -> Product:
+    return Product(
+        title=product.select_one(".title")["title"],
+        description=product.select_one(".description").text,
+        price=float(product.select_one(".price").text.replace("$", "")),
+        rating=len(product.select(".glyphicon-star")),
+        num_of_reviews=int(
+            product.select_one(".ratings > .pull-right").text.split()[0]
+        ),
+    )
+
+
+def get_products(url: str, driver: WebDriver) -> List[Product]:
+    driver.get(URLS[url])
+    try:
+        accept_cookies = driver.find_element(By.CLASS_NAME, "acceptCookies")
+        if accept_cookies:
+            accept_cookies.click()
+    except NoSuchElementException:
+        pass
+    try:
+        more_button = driver.find_element(
+            By.CLASS_NAME, "ecomerce-items-scroll-more"
+        )
+        while True:
+            more_button.click()
+    except Exception:
+        pass
```

Comment:
this is a bad practice, I think it is better to write something informative instead of `pass`